etc/scripts/util/test/cases/bad_compilers.properties

```diff
@@ -0,0 +1,2 @@
+compilers=a:b
+compiler.a.exe=/dev/null
```

Comment:
are the "no newline at end" of all these files intentional? (github gets angry)

Reply:
Nope, fixed :)